### PR TITLE
chore: [RC] cleanup default.json

### PR DESCRIPTION
**Pull request by MohamadJaara**
* remove unused values from KaliumConfigs

* clean up KaliumConfigs

* delete submitCrashReports

* delete FILE_RESTRICTION_LIST

(cherry picked from commit 195fc65cedab371ea13b3a88f068ff4397df2113)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

adding https://github.com/wireapp/wire-android-reloaded/pull/1306 to RC

### Causes (Optional)

_Briefly describe the causes behind the issues. This could be helpful to understand the adopted solutions behind some nasty bugs or complex issues._

### Solutions

_Briefly describe the solutions you have implemented for the issues explained above._

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1331/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5477689)(5477689)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5477689)(5477689)-dev-compat.apk)

File: default.json [hunk 1]
```diff
@@ -108,29 +74,13 @@
...
-    "wipe_on_cookie_invalid": false,
-    "force_private_keyboard": false,
-    "password_max_attempts": 0,
     "file_restriction_enabled": false,
     "file_restriction_list": "3gpp, aac, amr, avi, bmp, css, csv, dib, doc, docx, eml, flac, gif, html, ico, jfif, jpeg, jpg, jpg-large, key, m4a, m4v, md, midi, mkv, mov, mp3, mp4, mpeg, mpeg3, mpg, msg, ods, odt, ogg, pdf, pjp, pjpeg, png, pps, ppt, pptx, psd, pst, rtf, sql, svg, tex, tiff, txt, vcf, vid, wav, webm, webp, wmv, xls, xlsx, xml",
     "countly_server_url": "https://countly.wire.com",
     "countly_app_key": "79c8fc20713e0e877aa8d320ea078530604d3ea6",
     "force_constant_bitrate_calls": false,
-    "web_link_preview": true,
-    "keep_websocket_on": false,
```
Comment (ohassine) [inline default.json:133]: `keep_websocket_on` to remove also ?

**Review by ohassine — APPROVED** [review]
🧹✨

File: default.json (hunk 1 above)
Comment (MohamadJaara) [reply default.json:133]: yes it is not used anywhere

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1331/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5507413)(5507413)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5507413)(5507413)-dev-compat.apk)

**Pull request merged**